Sources/PackageRegistryTool/PackageRegistryTool+Publish.swift

```diff
@@ -133,7 +140,15 @@ extension SwiftPackageRegistryTool {
                     callbackQueue: .sharedConcurrent,
                     completion: callback
                 )
-            }
+            }*/
+            let publishConfiguration = RegistryClient.PublishConfiguration(
+                metadata: .init(location: [.archive, .request]), // FIXME
+                signing: .init(
+                    required: self.signingIdentity != nil || self.privateKeyPath != nil,
+                    acceptedSignatureFormats: [.CMS_1_0_0],
+                    trustedRootCertificates: []
+                )
+            )
 
             // step 2: generate source archive for the package release
             let metadataPath = self.customMetadataPath ?? packageDirectory.appending(component: Self.metadataFilename)
```

Comment:
In the latest spec:
- If `customMetadataPath` is set, send it in the request body.
- If `customMetadataPath` is not set, assume there is metadata file in package directory and it will be sent as part of the archive. SwiftPM doesn't copy any metadata file.

Reply:
thanks @yim-lee what should be a behavior if custom not specified and none is not found at the root of package?